scalardb/project.clj

```diff
@@ -8,8 +8,8 @@
                  [net.java.dev.jna/jna-platform "5.11.0"]
                  [org.slf4j/slf4j-jdk14 "2.0.6"]
                  [cassandra "0.1.0-SNAPSHOT"]
-                 [cc.qbits/alia "4.3.6"]
-                 [cc.qbits/hayt "4.1.0"]]
+                 [cheshire "5.12.0"]
+                 [com.scalar-labs/scalardb-schema-loader "3.10.1"]]
```

Comment:
Should we use 4.0.0-SNAPSHOT instead of the latest released version?

@brfrn169 What do you think?

Reply:
Ah, yes. I think so, too.